components/editor/modules/teasergroup/index.js

```diff
@@ -7,19 +7,21 @@ import {
   getIndex,
   getParentKey,
   insert,
-  move,
+  moveUp,
+  moveDown,
   remove
-} from '../teaser/dnd'
+} from '../teaser/actions'
 
 import { TeaserInlineUI } from '../teaser/ui'
 import { TeaserGroupButton, TeaserGroupForm } from './ui'
 
 export const getData = data => ({
```

Comment:
`createTeaserData`?

Reply:
otherwise unclear why get data in the group module says module `teaser`

Reply:
I'm not very happy with that solution either. This is all about giving nested groups and items some kind of deterministic context. I probably could have done it with teaserType, but since that gets copied to all child nodes of every teaser, it's not very helpful again.  I'll see